src/models/userDB.js

```diff
@@ -0,0 +1,14 @@
+const userDB = [{
```

Comment:
I think this file is not a model but a dummy DB, if you pull the latest master, you can move this into the src/dummy folder